oem.php

```diff
@@ -52,7 +52,7 @@
     </div>
     <div class="two-thirds copy">
         <h3 id="installation">Installation</h3>
-        <p>The elementary OS Installer treats every installation as an OEM installation; step through it to install the OS, then shut down and the device is ready to be shipped to a customer. To create a custom image, install the OS, configure the system with an administrator account, delete that account, then create an image of the disk. In both cases, Initial Setup will run on the customer's first boot and create their user account.</p>
+        <p>The elementary OS Installer treats every installation as OEM; step through it to install the OS, shut down the device and it is ready to be shipped to a customer. Create your own custom image by installing the OS, setting it up using an administrator account, deleting the account, and then creating an image of the disk. In both cases, Initial Setup is run upon the custoner's first boot, from which a user account is created.</p>
```

Comment:
Some of this is better but some is clunkier imho. How about:

```suggestion
        <p>The elementary OS Installer treats every installation as an OEM installation; step through it to install the OS, then shut down the device and it is ready to be shipped to a customer. Or, to create your own custom image: install the OS, configure any necessary packages or settings using a temporary administrator account, delete the account, and then create an image of the disk. In both cases, Initial Setup will create an end user account upon the customer's first boot.</p>
```